2021q3/sdhci_acpi.adoc

```diff
@@ -0,0 +1,19 @@
+=== Marvell SDHCI improvements and ACPI support
+
+Contact: Bartlomiej Grzesik <bag@semihalf.com> +
+Contact: Marcin Wojtas <mw@semihalf.com>
+
+The Semihalf team was working on the ACPI support for the
+link:https://www.marvell.com/content/dam/marvell/en/public-collateral/embedded-processors/marvell-infrastructure-processors-octeon-tx2-cn913x-product-brief-2020-02.pdf[Marvell Octeon TX2 CN913x] and link:https://www.marvell.com/content/dam/marvell/en/public-collateral/embedded-processors/marvell-embedded-processors-armada-7040-product-brief-2017-12.pdf[Armada 7k]/link:http://wiki.macchiatobin.net/tiki-index.php?page=Armada+8040[8k] SoCs'
+SD/MMC controller, as well as extending and improving the related generic parts of the
+FreeBSD OS.
+
+Applied changes:
+
+* Add ACPI_GET_PROPERTY to access Device Specific Data (DSD) link:https://cgit.freebsd.org/src/commit/?id=b91fc6c43a81d3b760fb570c8439a922e536d7e6[commit b91fc6c43a81]
+* Introduce generic device_get_property/device_has_property, which allow to obtain DT/ACPI properties with the same generic code link:https://cgit.freebsd.org/src/commit/?id=3f9a00e3b577dcca57e331842e0baf2dbdf9325f[commit 3f9a00e3b577]
+* Switch mmc_helper to device_* API, so that to access the controller description from DT/ACPI in a unified way link:https://cgit.freebsd.org/src/commit/?id=8a8166e5bcfb50e2b7280581b600d098fa6c9fc7[commit 8a8166e5bcfb]
```

Comment:
s/so that to/to/ (it would be "so as too", not "so that too", but just "to" is less verbose)